precompiles/ArbGasInfo_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021-2024, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package precompiles
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/offchainlabs/nitro/arbos/arbosState"
+	"github.com/offchainlabs/nitro/arbos/burn"
+	"github.com/offchainlabs/nitro/arbos/storage"
+	"github.com/offchainlabs/nitro/arbos/util"
+	"github.com/offchainlabs/nitro/util/testhelpers"
+)
+
+func TestArbGasInfo(t *testing.T) {
```

Comment:
This feels like it's actually checking several different calls in the same test. But, I can't tell if the results of the later calls actually depend on the results of the earlier calls. In many cases, it looks like they are unrelated. If so, they should be in separate test functions to make it clearer which part of the test is failing, and to make the setup for the test clearly connect to the assertions in the test.